### ISSUE TITLE
Seasons after Fall (366320)

### ISSUE THREAD

**Issue by NoXPhasma** [ValveSoftware/Proton#1846]
# Whitelist Request
- Name of the game to be whitelisted: Seasons after Fall
- Steam AppID of the game: 366320

## System Information
- GPU: GTX 970
- Driver/LLVM version: nvidia 396.54.09
- Kernel version: 4.18
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/NoXPhasma/9ed85e6b0602b656c812acd013ee121f
- Proton version: 3.16-3

## I confirm:
- [x] that pressing the `Play` button in the Steam client is sufficient.
- [ ] that [runtime config options](https://github.com/ValveSoftware/Proton#runtime-config-options)
      are necessary to run the game.
- [x] that no workarounds other than the mentioned ones are necessary.

## Issues
- [ ] I haven't experienced any issues.
- [ ] There are no issues left open for this game.
- [x] Although I consider the gaming experience equal to Windows there are
      remaining issues:

The only remain issue for me is that my gamepad (XBOX One S) doesn't work properly. In the main menu it will always scroll up and in the actual game it won't work at all. The game engine is Unity and it uses some obscure XInputInterface32.dll plugin. No workaround found to make my gamepad working (i.e. installing `xinput`).

**Comment by mimattr** [comment]
> The only remain issue for me is that my gamepad (XBOX One S) doesn't work properly. In the main menu it will always scroll

@NoXPhasma Agreed, the Steam Controller and official wired Xbox 360 Controller display the same behaviour. Both gamepads are detected but not working as expected with **Proton 3.16-6 Beta** (tested on Ubuntu 16.04 LTS)

[steam-366320.log](https://gist.github.com/mimattr/246966a771cb5bf2698368378bfbf56e)

[Steam System Information](https://gist.github.com/mimattr/b0b7f46366e31e254460b9593ce87e51)

**Comment by Skiski** [comment]
Since 3.16-8 beta, my xbox 360 gamepad works a little better. The controls with the stick works ok (with some lag I think) but the controls over the buttons is horrible. The jump button works once every 10 presses and it's very hard to do a high jump by keeping pressed the button.
With a keyboard, the controls are perfectly reactive and there is no issue.
With 4-2.1 beta, it's the same.

**Comment by NoXPhasma** [comment]
With Proton 4.11-3 my gamepad kinda works, but there is a very noticeable delay and many inputs are ignored.

**Comment by Skiski** [comment]
With 4.11-3 or 4.11-4, I have no more input lag with my xbox 360 controller. There is some lag, but that's due to the animations of the game taht are not interrupted by inputs. It's the same with a keyboard. It's most noticeable qhen you change direction. It's a feature, not a bug.

**Comment by joshas** [comment]
Working perfectly fine on Proton 5.13-4. Fully completed game using Xbox 360 controller.

**Comment by mimattr** [comment]
@joshas @Skiski Can also confirm the outstanding gamepad issue appears to be resolved with **Proton 5.13-4** (possibly earlier)

@NoXPhasma Could you retest to see if your issue is now resolved?

## System Information
- GPU: **RTX 2060**
- Driver/LLVM version: **NVIDIA 455.45.01**
- Distro version: **Ubuntu 20.04 (LTS)**
- [Link to full system information report as Gist](https://gist.github.com/mimattr/b0b7f46366e31e254460b9593ce87e51)
- Proton version: **5.13-4**

## I confirm:
- [x] that pressing the `Play` button in the Steam client is sufficient.

## Issues
- [x] I haven't experienced any issues.
- [x] There are no issues left open for this game.

**Comment by Skiski** [comment]
I can confirm it works perfectly with 5.13-4.
